src/main/java/commands/Command.java

```diff
@@ -0,0 +1,31 @@
+package commands;
+
+import tasklist.TaskList;
+
+/**
+ * Represents an abstract Command that should be inherited by
+ * Command subclasses. Should override the abstract methods to
+ * provide behavior expected of a Command class.
+ */
+abstract public class Command {
+    protected boolean exit = false;
```

Comment:
Should the boolean be called exit? Maybe you could use hasExit instead?
```suggestion
    protected boolean hasExit = false;
```